config/dev.webpack.config.js

```diff
@@ -19,7 +20,6 @@ const { config: webpackConfig, plugins } = config({
             changeOrigin: true,
             autoRewrite: true,
             ws: true,
-            ...['prod', 'stage'].includes(process.env.ENVIRONMENT) && { agent: new HttpsProxyAgent('http://squid.corp.redhat.com:3128') }
```

Comment:
Isn't this still required when user runs custom local API?

Reply:
Nope. That is already a part of the webpack proxy

Reply:
https://github.com/RedHatInsights/frontend-components/blob/master/packages/config-utils/proxy.js#L58

Reply:
Yeah, I know it's part of the general proxy. But previously there were errors when this config was missing from the customProxy as well. Is there any benefit of removing it?

Reply:
I don't have the local API setup. I would need someone who has to test it. I think having the proxy agent active twice might have some unwanted issues.

Reply:
Sure thing, let's remove it. If someone points to it that it won't work we'll re-add it.